pkg/networking/cilium/upgrade_plan.go

```diff
@@ -259,6 +259,20 @@ func ChangeDiff(currentSpec, newSpec *cluster.Spec) *types.ChangeDiff {
 
 func ciliumChangeDiff(currentSpec, newSpec *cluster.Spec) *types.ChangeDiff {
 	currentVersionsBundle := currentSpec.RootVersionsBundle()
+
+	newCiliumCfg := newSpec.Cluster.Spec.ClusterNetwork.CNIConfig.Cilium
+	if !newCiliumCfg.IsManaged() {
+		return &types.ChangeDiff{
+			ComponentReports: []types.ComponentChangeDiff{
+				{
+					ComponentName: "cilium",
+					OldVersion:    currentVersionsBundle.Cilium.Version,
+					NewVersion:    "Upgrade skipped (skipUpgrade: true)",
```

Comment:
NIT: I feel like it is not needed to specify, but I see merit in this too, so approving as is.
```suggestion
					NewVersion:    "Upgrade skipped",
```

Reply:
Yeah I thought it'd be better to state why the upgrade was skipped.